src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,50 @@
+package duke;
+
+import duke.command.Command;
+import duke.exception.DukeException;
+
+/**
+ * Represents a program called Duke. A <code> Duke </code> is a software which
+ * acts as a todolist.
+ */
+public class Duke {
+    private final Ui ui;
+    private final Storage storage;
+    private final TaskList tasks;
```

Comment:
I like the idea of the TaskList class, perhaps the name of the variable could be taskList instead ?